IPython/core/ultratb.py

```diff
@@ -688,7 +685,7 @@ def _format_exception_only(self, etype, value):
         have_filedata = False
         Colors = self.Colors
         list = []
-        stype = py3compat.cast_unicode(Colors.excName + etype.__name__ + Colors.Normal)
```

Comment:
In this case, I don't think it could ever have been bytes on Python 3, because `Colors.excName` will always be a str.